src/components/QueryBuilder.vue

```diff
@@ -1,27 +1,29 @@
 <template>
-	<div class="querybuilder">
-		<h1 class="querybuilder__heading">Simple Query Builder</h1>
+	<div class="querybuilder" role="main">
+		<h1 class="querybuilder__heading" role="heading" aria-level="1">Simple Query Builder</h1>
```

Comment:
I'm still not convinced. Doesn't https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role say that `<h1>` on its own is equivalent to another element with ` role="heading" aria-level="1"` set? Can't we just leave it as `<h1>`?

Reply:
oh that part, I read somewhere it's better to keep html5 and ARIA identifiers.
https://dequeuniversity.com/rules/axe/3.5/landmark-one-main
> It is a best practice to use both HTML 5 and ARIA landmarks to ensure all content is contained within a navigational region.

But OTOH, this is not a landmark. I'd remove the heading bit.